Update distribution.md (#415)
codemagic-ci-cd/codemagic-docs: content/publishing-yaml/distribution.md

echo $IOS_FIREBASE_SECRET | base64 --decode > $FCI_BUILD_DIR/ios/Runner/GoogleService-Info.plist
```

### Publishing an app to Firebase App Distribution

> Make sure to encrypt your Firebase token, it is possible to directly add it under publishing in the encrypted form, or encrypt `FIREBASE_TOKEN` as an environment variable and reference it under publishing. Check [documentation](https://firebase.google.com/docs/cli#cli-ci-systems) for details.

Reviewer: I'd start with an intro and add links to https://docs.codemagic.io/building/encrypting/ and the instructions to generating/obtaining the token itself, e.g.: To authenticate with Firebase, you need to create a Firebase token, encrypt it and save it to your Codemagic configuration. See how to obtain the token.

    android:
      groups: # Add one or more groups that you wish to distribute your Android application to, you can create groups in the Firebase console
        - androidTesters
        - ...
```

iOS

```yaml
publishing:
  firebase:
    firebase_token: Encrypted(...) # Add your encrypted Firebase token, or add it to your environment variables and reference as $FIREBASE_TOKEN
    ios:
      groups: # Add one or more groups that you wish to distribute your iOS application to, you can create groups in the Firebase console
        - iosTesters
        - ...
```

Android and iOS

```yaml
publishing:
  firebase:
    firebase_token: Encrypted(...) # Add your encrypted Firebase token, or add it to your environment variables and reference as $FIREBASE_TOKEN
    android:
      groups: # Add one or more groups that you wish to distribute your Android application to, you can create groups in the Firebase console
        - androidTesters
        - ...
    ios: # Add one or more groups that you wish to distribute your iOS application to, you can create groups in the Firebase console
      groups:
        - iosTesters
        - ...
```

> If you wish to pass release notes with your build, create a `release_notes.txt` file and add it to the project working directory, which is either the repository root directory or the Project path specified in the Build section in your workflow settings. Codemagic will fetch the content of that file and publish it with the build.

Reviewer: let's cross-reference with the release notes section

echo $IOS_FIREBASE_SECRET | base64 --decode > $FCI_BUILD_DIR/ios/Runner/GoogleService-Info.plist
```

### Publishing an app to Firebase App Distribution

Codemagic enables you to automatically publish your iOS or Android app to [Firebase Console](https://console.firebase.google.com/). Codemagic uses your **Firebase token** for authentication with Firebase App Distribution. To retrieve the token, follow the instructions in [Firebase documentation](https://firebase.google.com/docs/cli#cli-ci-systems).

> Make sure to [encrypt](https://docs.codemagic.io/building/encrypting/) your Firebase token. It is possible to add it under publishing in the encrypted form directly or encrypt `FIREBASE_TOKEN` as an environment variable and reference it under publishing.

Reviewer: ```suggestion
Make sure to [encrypt](https://docs.codemagic.io/building/encrypting/) your Firebase token. It is possible to directly add it under publishing in encrypted form, or you can encrypt it and save it to the `FIREBASE_TOKEN` environment variable and reference it under publishing.
```
